src/api/post/index.ts

```diff
@@ -2,7 +2,7 @@ import http from '@/api/core';
 import { CreatePost, SubTravelogue } from '@/types/post';
```

Comment:
proposal;

```suggestion
import { CreatePostType, SubTravelogueType } from '@/types/post';
```

`@types` 폴더에 정의한 타입의 타입 명칭 뒤에 Type이라고 명시해주면 어떨까요? 이 부분은 팀원이 의견이 다를 수 있으니 모든 팀원의 의견을 듣고 싶습니다. @kmww

Reply:
Icon과 마찬가지로 직관적이어서 좋은 것 같습니다!! 밬영하겠습니다.

Reply:
저도 좋은것같습니다!

Reply:
반영해두겠습니다~